docs/topics/templates/document-attributes.adoc

```diff
@@ -105,7 +105,7 @@ endif::[]
 :ProductDocVscGuideURL: https://access.redhat.com/documentation/en-us/{DocInfoProductNameURL}/{DocInfoProductNumber}/html-single/visual_studio_code_extension_guide
 :ProductDocIntelliJGuideURL: https://access.redhat.com/documentation/en-us/{DocInfoProductNameURL}/{DocInfoProductNumber}/html-single/intellij_idea_plugin_guide
 :OpenShiftDocsURL: https://docs.openshift.com/container-platform/{OpenShiftProductNumber}
-:LinkAPI: http://windup.github.io/windup/docs/latest/javadoc/
+:LinkAPI: https://github.com/windup/windup/blob/master/reporting/api/src/main/java/
```

Comment:
Here! JavaDoc is replaced

Please read comments properly. I said `Why have you comments this out on this line, but included in line 108`

Reply:
The link https://github.com/windup/windup/blob/master/reporting/api/src/main/java/org/jboss/windup/reporting/config/HintExists.java and the other links don't include the subdirectory `javadoc`  I changed the entire attribute.

The line I commented out refers to:

![image](https://github.com/windup/windup-documentation/assets/60698649/f1aaced7-9499-4fc6-aa6d-f439a8e2916e)

Clicking Javadox leads to a 404.

Reply:
@RichardHoch - Look at [docs/topics/rules-important-links.adoc](https://github.com/windup/windup-documentation/pull/893/files#diff-51e52bbc2015c9910f54e81a832c5d0e70cff62454cfb94910690a7634f32bcf)

Line 9 - you have commented out - `http://windup.github.io/windup/docs/latest/javadoc`

Line 108 - you have deleted - `http://windup.github.io/windup/docs/latest/javadoc/`
Line 108 - you have inserted - `https://github.com/windup/windup/blob/master/reporting/api/src/main/java/`

Note, originally line 9 and line 108 are the same link. You have replaced it in one instance but commented it out in the other.

So you have not been consistent in how you are handling the links. If you have an appropriate link then commenting it out on line 9 is incorrect, use the link from line 108.

Reply:
You have not fixed your mistake here

Reply:
Ah, you're suggesting I use {LinkAPI} as part of the address for Javadocs on line 8 of rules-important-links.adoc, right? OK.